### ISSUE TITLE
How many epoches should I set for my own dataset?

### ISSUE THREAD

**Issue by wen0618** [xingyizhou/CenterNet#434]
hello:
     I try to use CenterNet on my dataset. It has about 1200 images. I use 4 1080ti GUPS and batch size is 32.But after I trained 1000 epoches, the map is bad(10% less than yolo v3. about 20% map). I am poor in DL.So can you suggest me how many epoches should I set?Thank you so much!

**Comment by xingyizhou** [comment]
I usually train 70 epochs with learning rate dropped in the 60th epoch. You can visualize the output by add `--debug 2` to see if the dataset setup is correct.
